packages/foreman/rubygem-signet/rubygem-signet.spec

```diff
@@ -1,85 +1,101 @@
+# Generated from signet-0.11.0.gem by gem2rpm -*- rpm-spec -*-
+# template: scl
 %{?scl:%scl_package rubygem-%{gem_name}}
 %{!?scl:%global pkg_name %{name}}
 
 %global gem_name signet
 
-Summary: Signet is an OAuth 1.0 / OAuth 2.0 implementation
 Name: %{?scl_prefix}rubygem-%{gem_name}
-Version: 0.6.0
-Release: 7%{?dist}
+Version: 0.11.0
+Release: 1%{?dist}
+Summary: Signet is an OAuth 1.0 / OAuth 2.0 implementation
 Group: Development/Languages
-License: ASL 2.0
+License: Apache-2.0
 URL: https://github.com/google/signet/
 Source0: https://rubygems.org/gems/%{gem_name}-%{version}.gem
 
+# start specfile generated dependencies
 Requires: %{?scl_prefix_ruby}ruby(release)
-Requires: %{?scl_prefix_ruby}ruby(rubygems)
-Requires: %{?scl_prefix_ruby}ruby
+Requires: %{?scl_prefix_ruby}ruby >= 1.9.3
+Requires: %{?scl_prefix_ruby}ruby(rubygems) >= 1.3.5
 Requires: %{?scl_prefix}rubygem(addressable) >= 2.3
-Requires: %{?scl_prefix}rubygem(addressable) < 3.0
-Requires: %{?scl_prefix}rubygem(extlib) >= 0.9
-Requires: %{?scl_prefix}rubygem(extlib) < 1.0
+Requires: %{?scl_prefix}rubygem(addressable) < 3
 Requires: %{?scl_prefix}rubygem(faraday) >= 0.9
-Requires: %{?scl_prefix}rubygem(faraday) < 1.0
-Requires: %{?scl_prefix}rubygem(jwt) >= 1.0
-Requires: %{?scl_prefix}rubygem(jwt) < 2.0
+Requires: %{?scl_prefix}rubygem(faraday) < 1
 Requires: %{?scl_prefix_ror}rubygem(multi_json) >= 1.10
-Requires: %{?scl_prefix_ror}rubygem(multi_json) < 2.0
-
+Requires: %{?scl_prefix_ror}rubygem(multi_json) < 2
+Requires: %{?scl_prefix}rubygem(jwt) >= 1.5
+Requires: %{?scl_prefix}rubygem(jwt) < 3.0
 BuildRequires: %{?scl_prefix_ruby}ruby(release)
-BuildRequires: %{?scl_prefix_ruby}rubygems-devel
-BuildRequires: %{?scl_prefix_ruby}ruby
+BuildRequires: %{?scl_prefix_ruby}ruby >= 1.9.3
+BuildRequires: %{?scl_prefix_ruby}rubygems-devel >= 1.3.5
 BuildArch: noarch
 Provides: %{?scl_prefix}rubygem(%{gem_name}) = %{version}
-%{?scl:Obsoletes: ruby193-rubygem-%{gem_name}}
```

Comment:
I've been keeping these in until now, but I do wonder if we should start dropping these

Reply:
I'd nuke them once and for all, the `ruby193` SCL is so obsolete :)